src/api/python/cvc5.pxi

```diff
@@ -1229,15 +1372,19 @@ cdef class Solver:
     @expand_list_arg(num_req_args=0)
     def mkBitVector(self, *args):
         """
-        Supports the following arguments:
+            Create bit-vector value.
 
-        - ``Term mkBitVector(int size, int val=0)``
-        - ``Term mkBitVector(int size, string val, int base)``
+            Supports the following arguments:
 
-        :return: a bit-vector literal term
-        :param size: an integer size.
-        :param val: an integer representating the value, in the first form. In the second form, a string representing the value.
-        :param base: the base of the string representation (second form only)
+            - ``Term mkBitVector(int size, int val=0)``
```

Comment:
The implementation is weird here. Why not simply add `int size` as first argument and get rid of some of the manual checking?

Reply:
This PR is about docs, not the implementation. We should discuss this offline, but will not change any implementation in this PR.